### DISCUSSION TITLE
API Authentication

### DISCUSSION THREAD

**Discussion by rahulvemula** [rahulvemula/SE-leasingPortal#125] [Q&A]
What all apis need authentication to be accessed?

My proposal:

Register - doesnt need
Login - doesnt need. Obviously.

Apartments - doesnt need
Listings - doesnt need

Create/Get lease - will need
Update profile - will need
Create/update/Get complaint - will need

@vamsibvv @mitulmandaliya @lbarad Let me know your views

**Comment by mitulmandaliya** [comment]
Currently, I have added authorization for all endpoints except login and registration. I will remove it from the other endpoints based on the discussion here.
